packages/web/components/transactions/transaction-details.tsx

```diff
@@ -0,0 +1,174 @@
+import { Transition } from "@headlessui/react";
+import classNames from "classnames";
+import Image from "next/image";
+import { FunctionComponent } from "react";
+
+import { Icon } from "~/components/assets";
+import { CopyIconButton } from "~/components/buttons/copy-icon-button";
+import IconButton from "~/components/buttons/icon-button";
+import { Button } from "~/components/ui/button";
+import { ModalBase, ModalBaseProps } from "~/modals/base";
+
+export const TransactionDetailsContent = ({
+  onRequestClose,
+  isModal,
+}: {
+  onRequestClose: () => void;
+  isModal: boolean;
+}) => {
+  return (
+    <div
+      className={classNames(
+        "flex min-h-full flex-col",
+        !isModal && "border-l-[1px] border-osmoverse-700 bg-osmoverse-900"
+      )}
+    >
+      <div className={classNames("mx-4 flex flex-col", !isModal && "fixed")}>
+        {!isModal && (
+          <div className="py-4">
+            <IconButton
+              aria-label="Close"
+              mode="unstyled"
+              size="unstyled"
+              className="w-fit cursor-pointer py-0 text-osmoverse-400 hover:text-white-full"
+              icon={<Icon id="close" width={48} height={48} />}
+              onClick={onRequestClose}
+            />
+          </div>
+        )}
+        <div className="flex flex-col items-center gap-4 pt-2 pb-6">
+          <div className="flex h-12 w-12 items-center justify-center rounded-full bg-osmoverse-825">
+            <Icon id="swap" width={24} height={24} aria-label="swap icon" />
+          </div>
+          <div className="flex flex-col items-center justify-center gap-2 text-center">
+            <div className="text-h5">Swapped</div>
+            <div className="text-body1 text-osmoverse-300">
+              March 14, 2024, 13:01
+            </div>
+          </div>
+        </div>
+        <div className="flex flex-col rounded-2xl border border-osmoverse-700 p-2">
+          <div className="flex justify-between p-2">
+            <div className="flex gap-4">
+              <Image
+                alt="OSMO"
+                src="/tokens/generated/osmo.svg"
+                height={32}
+                width={32}
+              />
+              <div className="flex flex-col">
+                <div className="text-subtitle1">Sold</div>
+                <div className="text-body1 text-osmoverse-300">OSMO</div>
+              </div>
+            </div>
+            <div className="flex-end flex flex-col text-right">
+              <div className="text-subtitle1">$100.00</div>
```

Comment:
These could be `subtitle1` instead of `text-subtitle1` so we also apply the font. I see many of these across the file.